papermill/preprocess.py

```diff
@@ -67,20 +69,33 @@ def papermill_process(self, nb_man, resources):
         return nb, resources
 
     def log_output_message(self, output):
+        """
+        Process a given output. May log it in the configured logger and/or write it into
+        the configured stdout/stderr files.
+
+        :param output: nbformat.notebooknode.NotebookNode
+        :return:
+        """
         if output.output_type == "stream":
+            content = "".join(output.text)
             if output.name == "stdout":
-                self.log.info("".join(output.text))
+                if self.log_output:
+                    self.log.info(content)
+                if self.stdout_file:
+                    self.stdout_file.write(content)
```

Comment:
You'll want to do `.encode('utf-8')` if it's python 2 (python 3 is fine as is).

You can see the issue by adding this to a cell:
```
print('😀')
```